From 0d789784a27856a10c331153c93d6aff57005cac Mon Sep 17 00:00:00 2001
From: dankoman <dankoman30@gmail.com>
Date: Tue, 10 Feb 2015 20:37:15 +0100
Subject: [PATCH 08/26] Layers : fix sensitive lockscreen notification title
 and time

fix this crap when no overlay is being utilized:
http://i.imgur.com/XjW5igU.png

Change-Id: I5e10e5d4bb4249278649fa9c58120932c2dd7195
---
 packages/SystemUI/src/com/android/systemui/statusbar/BaseStatusBar.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/BaseStatusBar.java b/packages/SystemUI/src/com/android/systemui/statusbar/BaseStatusBar.java
index 14dff4e..f811d48 100755
--- a/packages/SystemUI/src/com/android/systemui/statusbar/BaseStatusBar.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/BaseStatusBar.java
@@ -1437,6 +1437,7 @@ public abstract class BaseStatusBar extends SystemUI implements
             } catch (NameNotFoundException e) {
                 title.setText(entry.notification.getPackageName());
             }
+            title.setTextColor(R.color.exposed_primary_text_light);
 
             final ImageView icon = (ImageView) publicViewLocal.findViewById(R.id.icon);
             final ImageView profileBadge = (ImageView) publicViewLocal.findViewById(
@@ -1481,6 +1482,7 @@ public abstract class BaseStatusBar extends SystemUI implements
                 time.setVisibility(View.VISIBLE);
                 time.setTime(entry.notification.getNotification().when);
             }
+            time.setTextColor(R.color.exposed_primary_text_light);
 
             final TextView text = (TextView) publicViewLocal.findViewById(R.id.text);
             if (text != null) {
-- 
1.9.1

